archive_converter.py

```diff
@@ -67,7 +67,7 @@ def _add_messages_to_index(self, new_messages : List[Message]):
         for message in need_parent:
             parent = self._message_dao.get(message.in_reply_to)
             if not parent:
-                print("Could not find parent email, dropping " + message.subject)
+                logging.info('Could not find parent email, dropping %s', message.id)
```

Comment:
Might want to make a function which prints out the relevant details for dropping an email since you log that message in a couple places.

Probably want to include:
1) message ID as you have done
2) commit hash from lore (since it is hard to search by message ID in lore git repos (if I remember correctly)
3) list the message ID as a web address link which will take you to the email in lore.